embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/network/logging/EmbraceNetworkLoggingService.kt

```diff
@@ -166,88 +168,66 @@
      * @param networkCall that is going to be captured
      */
     private fun processNetworkCall(callId: String, networkCall: NetworkCallV2) {
-        // Get the domain, if it can be successfully parsed
+        // Get the domain, if it can be successfully parsed. If not, don't log this call.
         val domain = networkCall.url?.let {
             getDomain(it)
-        }
-
-        if (domain == null) {
-            logger.logDeveloper("EmbraceNetworkLoggingService", "Domain is not present")
-            return
-        }
-
-        logger.logDeveloper("EmbraceNetworkLoggingService", "Domain: $domain")
+        } ?: return
 
         if (isIpAddress(domain)) {
-            logger.logDeveloper("EmbraceNetworkLoggingService", "Domain is an ip address")
-            val captureLimit = configService.networkBehavior.getNetworkCaptureLimit()
-
-            if (ipAddressCount.getAndIncrement() < captureLimit) {
-                // only capture if the ipAddressCount has not exceeded defaultLimit
-                logger.logDeveloper("EmbraceNetworkLoggingService", "capturing network call")
+            if (ipAddressNetworkCallCount.getAndIncrement() < defaultPerDomainCallLimit) {
                 storeNetworkCall(callId, networkCall)
-            } else {
-                logger.logDeveloper("EmbraceNetworkLoggingService", "capture limit exceeded")
             }
             return
+        } else if (!domainSetting.containsKey(domain)) {
+            createLimitForDomain(domain)
```

Comment:
Added the fetching of the domain limits here to run before we record a network request. It was doing it after and I have no idea why.